app/models/ability.rb

```diff
@@ -84,45 +86,52 @@ def initialize_wimi(user)
 
   def initialize_representative(user)
     initialize_wimi user
-    can :read, Holiday do |h|
-      user.is_representative?(h.user.chair)
-      h.status != ('saved' || 'declined')
-    end
+
     can :see_holidays, User do |chair_user|
       chair_user.chair == user.chair
     end
+    can :see_trips,    User do |chair_user|
+      chair_user.chair == user.chair
+    end
+
     can :reject, Holiday.select {|h| h.user != user}
     can :accept, Holiday.select {|h| h.user != user}
-    can :read, Trip.select { |t| user.is_representative?(t.user.chair) }
-    can :read, TravelExpenseReport.select { |t| user.is_representative?(t.user.chair) }
-    can :read, Chair do |chair|
-      user.is_representative?(chair)
+    can :read,   Holiday do |h|
+      user.is_representative?(h.user.chair)
+      h.status != ('saved' || 'declined')
     end
-    can :requests, Chair do |chair|
-      user.is_representative?(chair)
+
+    can :read,      Trip.select { |t| user.is_representative?(t.user.chair) }
+    can :reject,    Trip
+    can :accept,    Trip
+    can :edit_trip, Trip do |trip|
+      trip.status != 'saved'
     end
-    can :read, Trip do |t|
+    can :read,      Trip do |t|
       user.is_representative?(t.user.chair)
       t.status != ('saved' || 'declined')
     end
-    can :see_trips, User do |chair_user|
-      chair_user.chair == user.chair
+
+    can :read, TravelExpenseReport.select { |t| user.is_representative?(t.user.chair) }
```

Comment:
TravelExpenseReport wurde in Expense umbenannt. Die Changes gibts am vollstaendigsten im PR #270.